spec/models/user_spec.rb

```diff
@@ -297,4 +297,25 @@
       expect { do_purge }.to change(User, :count).by(0)
     end
   end
+
+  #sam Unit test
+  describe User do
+    let(:emails) {  ['a@a.com','b@b.com'] }
+    let(:users) { double('Users') }
+        it '.superadmin_emails' do 
+          expect(users). to receive(:pluck).with(:email).and_return(emails)
+          expect(User).to receive(:superadmins).and_return(users)
+          expect(User.superadmin_emails).to eq emails
+        end
+  end
+
+  #integration test 
```

Comment:
Hmmm. The method from where we extracted `superadmin_emails` is `send_email_to_superadmin_about_org_edit`. So I am trying to replicate this. But I CAN generate an email to superadmin when I `create` a new org but not when somebody makes a `org edit`. What's the workflow so when I login as superadmin and go to the "Admin" tab and I click "Pending Proposed edits" I see an email for approval. Makes sense?